### PR TITLE
[Refactor] 基于 v2.x 重构 v3.x

### LINKED ISSUE #32: 摘要部分的字体问题
参考官方的Word模版，Abstract应该是Times New Romance字体？

### LINKED ISSUE #33: 论文英语部分字体问题
显示出来的pdf文件英语部分字体是LMRoman12-Regular-Identity-H，有没有办法修改成times new roman？

### PR DESCRIPTION
**Pull request by jamesnulliu**
## Updates

- 重构项目目录.
- 增加 bash 编译脚本.
- 增加 CD: Release.

## Bug Fixes

- Fixes #33 
- Fixes #32

### CODE REVIEW

**Comment by Errant404** [comment]
LGTM!

File: scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+MAINTEX_NAME="main"
+
+show_usage() {
```
Comment (Errant404) [inline scripts/build.sh:6]: If your sh script includes a Shebang, you should not execute it using `bash build.sh`.

Comment (jamesnulliu) [reply scripts/build.sh:6]: I believe while it's true that a shebang specifies which interpreter should execute the script, there's nothing wrong with explicitly using bash to run a script that includes a shebang line. Using bash script.sh will work perfectly fine - it just bypasses the shebang specification.

Comment (Errant404) [reply scripts/build.sh:6]: That's certainly not a problem; I just prefer the UNIX style. 😂

**Comment by Errant404** [comment]
@LuminolT, do you have any questions? If not, I will proceed with the Squash merge.

**Review by LuminolT — APPROVED** [review]
I have further reviewed the related issues.

Regarding the first problem, although this modification approach will increase the difficulty of subsequent low-level configurations, overall, it will work, and there are no issues.

As for the second one, based on the GB/T 7713.1-2006 national standard, changing to full-width characters is perfectly acceptable.

@Errant404 please go ahead and merge this PR if there are no further issues. THX!

**Pull request merged**